packages/monitoring-system/metrics-dashboard/src/app.html

```diff
@@ -0,0 +1,95 @@
+<html>
```

Comment:
This is just the old `index.html` renamed to `app.html` so that `index.html` can contain the authentication workflow